tests/security/selinux/selinux_setup.pm

```diff
@@ -32,7 +32,16 @@ sub run {
     }
 
     # program 'sestatus' can be found in policycoreutils pkgs
-    zypper_call("in policycoreutils policycoreutils-python");
+    zypper_call("in policycoreutils");
+    my $ret = script_run('zypper -n in policycoreutils-python');
+    if ($ret) {
+        if ($ret == 104) {
```

Comment:
it could be like this `if (defined $ret && $ret == 104)`

Reply:
done, thanks for the pointing.